### PR TITLE
List AnsibleTower Inventories

### PR DESCRIPTION
**Pull request by ColeHiggins2**
**Description:**

With the incoming requirement to have broker accept and pass along inventory information, it would be nice to let users be able to figure out what queries are available.

Desired usage:
Get a list of inventories your account has access to.

**broker providers AnsibleTower --inventories**

Optional usage:
Get some relevant information about an inventory.

**broker providers AnsibleTower --inventory test-inventory**

Test Results:

Screenshot from 2021-02-05 15-04-09

### CODE REVIEW

**Comment by ColeHiggins2** [comment]
Fixed Inventory output results:


Screenshot from 2021-02-05 16-00-14

**Review by JacobCallahan — APPROVED** [review]
ACK, nice addition!

Fixes #84

File: broker/commands.py [hunk 1]
```diff
@@ -57,8 +57,9 @@ def provider_cmd(*args, **kwargs):  # the actual subcommand
             if prov_info[0] == prov_class
         ):
             action = action.replace("_", "-")
+            plural = action.replace('y', 'ies') if action.endswith('y') else f"{action}s"
```
Comment (mshriver) [inline broker/commands.py:60]: I'm suspect of this change, we now have to be careful about adding provider actions that end in `y`.

Consider adding `inflect` as a requirement and using it for this conversion.

**Review by mshriver — APPROVED** [review]

File: broker/commands.py (hunk 1 above)
Comment (JacobCallahan) [reply broker/commands.py:60]: Let's cross that bridge if we come to it. I'd like to not introduce more dependencies if they aren't currently necessary.

**Pull request merged**